android/src/main/java/com/opentokreactnative/OTSessionManager.java

```diff
@@ -421,6 +424,16 @@ public void setVideoTransformers(String publisherId, ReadableArray videoTransfor
         }
     }
 
+    @ReactMethod
+    public void setAudioTransformers(String publisherId, ReadableArray audioTransformers) {
+        ConcurrentHashMap<String, Publisher> mPublishers = sharedState.getPublishers();
+        Publisher mPublisher = mPublishers.get(publisherId);
+        if (mPublisher != null) {
+          ArrayList<AudioTransformer> nativeAudioTransformers = Utils.sanitizeAudioTransformerList(mPublisher, audioTransformers);
```

Comment:
@jeffswartz should we run an exception if list is empty?

Reply:
@marinaserranomontes No. Passinig in an empty array is the way to clear audio transformers (per the docs).